Formula/lima.rb

```diff
@@ -0,0 +1,33 @@
+class Lima < Formula
+  desc "Linux virtual machines"
+  homepage "https://github.com/AkihiroSuda/lima"
+  url "https://github.com/AkihiroSuda/lima.git",
+    tag:      "v0.2.1",
```

Comment:
I'll release v0.3.0 in a few hours

Reply:
Released v0.3.0 https://github.com/AkihiroSuda/lima/releases/tag/v0.3.0

Reply:
Brew file was updated